include/wx/msw/chkconf.h

```diff
@@ -183,7 +183,7 @@
     #define wxUSE_SECRETSTORE 0
 #endif
 
-#if !wxUSE_OWNER_DRAWN && !defined(__WXUNIVERSAL__)
+#if !wxUSE_OWNER_DRAWN && !defined(__WXUNIVERSAL__) && defined(__WXMSW__)
```

Comment:
I think this file implicitly assumes being used only for wxMSW, so it would be nice to make it more clear that it can now be used with the other ports too, e.g. like this:

    /* wxMSW-specific checks: notice that this file is also used with wxUniv and
       can even be used with wxGTK, when building it under Windows. */
    #if defined(__WXMSW__) && !defined(__WXUNIVERSAL__)
         ....
    #endif